### PR TITLE
Implement behat tests

### LINKED ISSUE #132: Add functional tests
The priority is to test :

- [x] User register
- [x] User connection
- [x] User availabilities
- [x] Organization connection
- [ ] Organization planning filters
- [ ] Organization availability book

### PR DESCRIPTION
**Pull request by vincentchalamon**
Fixes #132

- [x] Install & configure Behat
- [x] Add User login test
- [x] Add User register test
- [x] Add Organization login test

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -119,7 +119,10 @@ jobs:
             - uses: codecov/codecov-action@v1
               with:
                 token: ${{ secrets.CODECOV_TOKEN }}
-                
+
+            - run: vendor/bin/behat
```
Comment (mRoca) [inline .github/workflows/ci.yml:123]: can you add it in `make test` ?

File: tests/Behat/SecurityContext.php [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    {
+        /** @var InitializedContextEnvironment $environment */
+        $environment = $scope->getEnvironment();
+        /** @var MinkContext $minkContext */
+        $minkContext = $environment->getContext(MinkContext::class);
+        $this->minkContext = $minkContext;
+    }
+
+    /**
+     * @Given I am authenticated as a user
```
Comment (mRoca) [inline tests/Behat/SecurityContext.php:48]: maybe you can allow to choose the user : `Given I am a user authenticated as "\w+"`, and `Given I am an organization authenticated as "\w+"`,

Comment (mRoca) [reply tests/Behat/SecurityContext.php:48]: `Given I am a user authenticated as "user1"`, `Given I am an organization authenticated as "UL 05"`

Comment (vincentchalamon) [reply tests/Behat/SecurityContext.php:48]: Already done on line 64:
```
I am authenticated as "user1@resop.com"
I am authenticated as "UL 05"
```

**Pull request merged**